rir/src/runtime/Function.h

```diff
@@ -46,63 +46,45 @@ typedef SEXP FunctionSEXP;
  */
 #pragma pack(push)
```

Comment:
Unrelated, but...are these pragmas still needed?

Reply:
Hmm, I'm not sure. I tried compiling with and without, and with my compiler, the pragma saves us 4 bytes in every `Function` object.

Here's the layout I observed:

```c++
struct Function {
  // 0 bytes from start
  // struct RirRuntimeObject {
  //   struct rir_header {
  //     uint32_t gc_area_start;
  //     uint32_t gc_area_end;
  //     uint32_t magic;
  //   };
  // }; // 12 bytes

  // 12 bytes from start
  unsigned size; // 4 bytes

  // 16 bytes from start
  FunctionSignature* signature; // 8 bytes

  // 4 bytes padding (can't remove with #pragma pack)

  // 28 bytes from start
  unsigned deopt : 1;
  unsigned markOpt : 1;
  unsigned spare : 30;  // 4 bytes

  // 32 bytes from start
  unsigned codeLength; // 4 bytes

  // 4 bytes padding (removed with #pragma pack)

  // 40 bytes from start (36 with #pragma pack)
  FunctionSEXP origin_; // 8 bytes

  // 48 bytes from start (44 with #pragma pack)
  FunctionSEXP next_; // 8 bytes

  CodeSEXP codeObjs[]; // 0 bytes (dynamically allocated)
}; // end is 56 bytes from start (52 bytes with #pragma pack)
```